.eslintrc.js

```diff
@@ -37,7 +36,7 @@ module.exports = {
     ecmaFeatures: {
       jsx: true,
     },
-    ecmaVersion: 2018,
+    ecmaVersion: "latest",
```

Comment:
is this the equivalent of esnext?

Reply:
I think so 🤔 ([docs](https://eslint.org/docs/latest/user-guide/configuring/language-options))

<img width="1098" alt="Screen Shot 2022-07-12 at 12 42 34 PM" src="https://user-images.githubusercontent.com/47064971/178547313-2fd878a1-312c-44e5-9bf1-88a78403c9e8.png">